docker-compose-test.yml

```diff
@@ -0,0 +1,133 @@
+# todo: simplify or load minimum test info needed from local files 
+# for teardown and startup and to remove volumes
+
+services:
+
+  httpd:
```

Comment:
Redefining the services like this might be overkill, but using `extends` and pulling in pre-defined services that way still uses the defined volumes and depends services